### PR TITLE
pdf button

### PR DESCRIPTION
**Pull request by TimCsaky**

### CODE REVIEW

**Comment by TimCsaky** [comment]
seems to be working. not sure if the pdf will populate once we update the word template

File: app/frontend/src/components/form/Step6.vue [hunk 1]
```diff
@@ -191,6 +192,30 @@ export default {
     downloadPdf() {
```
Comment (loneil) [inline app/frontend/src/components/form/Step6.vue:192]: Can chop this unused

File: app/frontend/src/components/form/Step6.vue [hunk 2]
```diff
@@ -191,6 +192,30 @@ export default {
...
+      ipcService
+        .getIPCContentAsPDF(this.submissionDetails.ipcPlan.ipcPlanId)
+        .then(response => {
+          const blob = new Blob([response.data], {
+            type: 'attachment'
+          });
+          this.createDownload(blob, `${this.submissionDetails.ipcPlan.ipcPlanId}.pdf`);
+        })
+        .catch(() => {
+          this.showTableAlert('error', 'Currently unable to complete this request.');
```
Comment (loneil) [inline app/frontend/src/components/form/Step6.vue:205]: This function isn't in this component

File: app/src/routes/v1/ipc.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ router.get('/:ipcPlanId', keycloak.protect(), async (req, res, next) => {
   }
 });
 
-router.get('/pdf/:ipcPlanId', keycloak.protect(), async (req, res, next) => {
+router.get('/pdf/:ipcPlanId', async (req, res, next) => {
```
Comment (loneil) [inline app/src/routes/v1/ipc.js:54]: Discuss after getting this working with Jason if we want to rate limit this endpoint now

File: app/frontend/src/components/form/Step6.vue (hunk 1 above)
Comment (TimCsaky) [reply app/frontend/src/components/form/Step6.vue:192]: doing it now 👍

File: app/frontend/src/components/form/Step6.vue [hunk 4]
```diff
@@ -213,8 +214,31 @@ export default {
...
+        .then(response => {
+          const blob = new Blob([response.data], {
+            type: 'attachment'
+          });
+          this.createDownload(blob, `${this.submissionDetails.ipcPlan.ipcPlanId}.pdf`);
+        })
+        .catch(() => {
+          /*eslint-disable */
+          console.log('Currently unable to complete this request.');
+          /*eslint-enable */
```
Comment (jujaga) [inline app/frontend/src/components/form/Step6.vue:229]: console.log('Currently unable to complete this request.'); // eslint-disable-line no-console

Comment (TimCsaky) [reply app/frontend/src/components/form/Step6.vue]: done

**Pull request merged**